languages/javascript/scripts/scaffold/template/babel.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
Is this the exact same we already use? If yes, you can just thumbs up this and resolve.

I don't think `useBuiltIns` should be `false`.